Tasks/AzureResourceManagerTemplateDeploymentV3/main.ts

```diff
@@ -0,0 +1,45 @@
+import tl = require("azure-pipelines-task-lib/task");
+import path = require("path");
+
+import armDeployTaskParameters = require("./models/ARMDeployTaskParameters");
+import resourceGroup = require("./operations/ResourceGroup");
+import armResource = require("azure-arm-rest-v2/azure-arm-resource");
+import armManagementGroup = require("azure-arm-rest-v2/azure-arm-management-group");
+import armSubscription = require("azure-arm-rest-v2/azure-arm-subscription");
+import { DeploymentParameters } from "./operations/DeploymentParameters";
+import { DeploymentScopeBase } from "./operations/DeploymentScopeBase";
+
+function run(): Promise<void> {
+    var armTemplateDeploymentTaskParameters = new armDeployTaskParameters.ARMDeployTaskParameters();
+    return armTemplateDeploymentTaskParameters.getARMTemplateDeploymentTaskParameters().then((taskParameters) => {
+        if(taskParameters.deploymentScope === "Management Group"){
+            var deploymentParameters = new DeploymentParameters({}, taskParameters.location);
+            var managementGroupOperationsController = new DeploymentScopeBase(new armManagementGroup.ManagementGroupManagementClient(taskParameters.credentials, taskParameters.managementGroupId), taskParameters, deploymentParameters);
```

Comment:
spread the arguments across multiple lines for readability purpose

Reply:
Done